File: palindromeRecursive.js
const palindrome = (string) => {  
  string = string.toLowerCase().split("").join('')
  if (string.length <= 1){
    return true
  } else if (string[0] != string[string.length -1]){
    return false
  }
  console.log(palindrome(string.slice(1,-1)))
  return palindrome(string.slice(1,-1))
}
console.log(palindrome('civic'))

File: palindrome_recursive.py
def palindrome(string):  
  string = str(string)
  string = ''.join(e for e in string if e.isalnum()).lower()
  if len(string) <= 1:
    return True
  elif string[0] != string[-1]:
    return False
  return palindrome(string[1:-1])
print(palindrome('Civic'))
